.markdownlint.json

```diff
@@ -0,0 +1,33 @@
+{
+    "_docs_for_this_file": "https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md",
+    "line-length": false,
+    "first-line-h1": true,
```

Comment:
i'd rather have this be false and not have to disable and enable things in the README...

Reply:
absolutely agree